### ISSUE TITLE
Export information to PDF

### ISSUE THREAD

**Issue by hectorsaanchez** [TheAarxn/GESPRO_Practica_2_Curso_2024_2025#41]
Añade la funcionalidad de exportar reportes o estados financieros en formato PDF.

**Comment by Dankof04** [comment]
Deberíamos darle al usuario diferentes opciones de exportación de forma simplificada o sería más conveniente hacer un menú avanzado para que cada usuario elija sus preferencias acorde a sus conocimientos?

**Comment by Dankof04** [comment]
En la planificación del sprint hemos acordado hacer el menú avanzado :D

**Comment by Dankof04** [comment]
Exacto, le ofreceremos al usuario distintos modos de interacción y él mismo elegirá la complejidad de su interfaz de exportación

**Comment by Dankof04** [comment]
De acuerdo, duda aclarada
